src/main/java/com/revature/domain/Eval.java

```diff
@@ -58,7 +61,7 @@
 	public Eval() {/*empty constructor needed*/}
 
 	public Eval(Integer week, Date date, EvalType evalType, Person trainee, Batch batch,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS00107)